src/components/ChildFormFragment.tsx

```diff
@@ -130,11 +135,11 @@ const ChildFormFragment: React.FC<{ field: FormListFieldData }> = ({
 
       <Form.Item
         {...fieldWithoutKey}
-        name={[field.name, 'picture']}
-        fieldKey={[field.fieldKey, 'picture']}
+        name={[field.name, 'profilePicture']}
+        fieldKey={[field.fieldKey, 'profilePicture']}
         label="Upload Profile Picture"
       >
-        <Dragger>
+        <Dragger multiple={false} beforeUpload={() => false}>
```

Comment:
What do these props do?

Reply:
Is this beforeUpload meant to be a no-op then? If so we should just use lodash's noop util